content/en/events/kcsna2021/registration.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Registration"
+type: docs
+weight: 2
+---
+
+### Attend in-person
+
+**NOTE:** The Kubernetes Contributor Summit will follow KubeCon’s
+[Health & Safety Guidelines] requiring **ALL** in-person attendees to be fully
+vaccinated against the COVID-19 virus.
+
+To attend in person you must be a member of one of the Kubernetes GitHub Orgs
+(e.g. [kubernetes] or [kubernetes-sigs]) or a sponsored attendee. If you are an
```

Comment:
@mrbobbytables Extend to kubernetes-csi and kubernetes-client org ?

Reply:
I didn't want to add all of the there, but might be worth calling out. Can definitely do it when we publish it with reg links.
All of them are enumerated in the faq though.